src/pages/profile/index.js

```diff
@@ -228,43 +224,44 @@ const ViewProfile = () => {
               <div className="contact-info-content info-grid">
                 <h4 className="padding-title">Contact info</h4>
                 <div className="padding-top"></div>
-                <small className="padding-field-name">Email*</small>
+                <small className="padding-field-name">Email</small>
                 <textarea
                   cols="40"
                   className="textarea-small"
                   placeholder="alex.walker@boolean.co.uk"
                   disabled
                   value={userProfile.email}
-
                 ></textarea>
                 <div className="padding"></div>
-                <small className="padding-field-name">Mobile*</small>
+                <small className="padding-field-name">Mobile</small>
                 <textarea
                   cols="40"
                   className="textarea-small"
                   placeholder="07890 123456"
                   disabled
                 ></textarea>
                 <div className="padding"></div>
-                <small className="padding-field-name">Password*</small>
-                <div class="input-container">
-                  <div className="password-field">
-                    <textarea
-                      rows="10"
-                      cols="40"
-                      class="textarea-small"
-                      name="password"
-                      placeholder="***************"
-                      disabled
-                      required
-                    ></textarea>
-                    <EyeIcon className="eye-icon"/>
-                  </div>
-                </div>
-
-                <div className="padding-bottom"></div>
-                <br></br>
-                <small>*Required</small>
+                {
+                  ( userId === userProfile.id ) ? (
+                    <>
+                      <small className="padding-field-name">Password*</small>
+                      <div class="input-container">
+                        <div className="password-field">
+                          <textarea
+                            rows="10"
+                            cols="40"
+                            class="textarea-small"
+                            name="password"
+                            placeholder="***************"
+                            disabled
+                            required
+                          ></textarea>
+                          <EyeIcon className="eye-icon"/>
+                        </div>
+                      </div>
+                    </>
+                  ) : null
```

Comment:
If you want to show something or not, use an && statement.

If you want to show something or something else, use an if statement